client/style.css

```diff
@@ -35,115 +35,123 @@ h1{
   font-family: Helvetica, Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
 }
 
+h2 {
```

Comment:
If we style the basic HTML types, we should make sure that everybody knows that it will be styled this way. IMO the types like h1-6, p, span, etc. should be left alone unless everyone is in agreement about what they want it to do. The thing that's striking me is the gray color, as I'm not sure everyone wants the second header to be gray.

Reply:
And same with padding.

Reply:
I am a fan of setting styles for these early in the project, actually, so we can all get used to it. Restyling the base components to have a project-default that we all get used to is a positive in my book. While it's not "early in the project" relative to this project's real scope... well, it's been less than 3 weeks! In real world, it's still early.

Reply:
Thank you for the correction on this, I fixed and pushed the commit.